.travis.yml

```diff
@@ -39,7 +39,7 @@ install:
       fi
 
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-        brew uninstall --ignore-dependencies libjpeg libtiff;
+        brew uninstall --ignore-dependencies gdal giflib libjpeg libtiff;
```

Comment:
Why are we getting these errors now and not in core tiledb? Is travis here using different versions of osx/ubuntu?

Reply:
I don't really know, currently I'm assuming it's some bad cached state on the worker image (hopefully if I can fix it I can then remove these explicit brew steps from the travis file).